.docker/subzero.Dockerfile

```diff
@@ -44,14 +44,15 @@ RUN mv /usr/share/ca* /tmp && \
 	ln -s /subzero/.local/share/subzero /data
 
 COPY --from=builder /subzero/target/$PROFILE/subzero /usr/local/bin
+COPY --from=builder /subzero/.docker/chainspec /chainspec
 
 # checks
 RUN ldd /usr/local/bin/subzero && \
 	/usr/local/bin/subzero --version
 
 # Shrinking
-RUN rm -rf /usr/lib/python* && \
-	rm -rf /usr/bin /usr/sbin /usr/share/man
+# RUN rm -rf /usr/lib/python* && \
```

Comment:
I am wondering what was the reason to execute this lines, @2075?
It doesn't seem to reduce image size significantly.

Reply:
efficiency and less clutter on the image

Reply:
I've commented it because it removes shell interpreters and I can't run entrypoint.sh.

Reply:
it is removing non required entrypoints, feel free to add what you need but a node image should be single minded, no?